### ISSUE TITLE
Add consultation submissions to the DB for later processing

### ISSUE THREAD

**Issue by sofiaromorales** [setlife-network/setlife-solutions#193]

**Comment by Ovifer13** [comment]
@sofiaromorales @otech47 for this we have to create new tables in the DB ?

**Comment by sofiaromorales** [comment]
yes, but I still need to think about which is the best DB design for this, if you have any idea we can discuss it here

**Comment by Ovifer13** [comment]
Where can I see the DB Model? @sofiaromorales

**Comment by sofiaromorales** [comment]
I drafted this a while ago https://github.com/setlife-network/setlife-solutions/issues/50#issuecomment-1159979086 but might be outdated, you should be able to understand the model from the codebase

**Comment by sofiaromorales** [comment]
`consultation` table
* id
- name
- email
- phone_number
- company_type
- min_budget
- max_budget
- project_goals
- description
+ constrains
